utils/constituencyData.ts

```diff
@@ -24,38 +16,16 @@ const getConstituencyData = async (
   return constituencyData;
 };
 
-const getConstituencySlugs = async (
-  cached: boolean = true,
-): Promise<string[]> => {
-  const constituenciesData = await getConstituenciesData(cached);
+const getConstituencySlugs = (cached: boolean = true): string[] => {
+  const constituenciesData = getConstituenciesData(cached);
   return constituenciesData.map(
     (c: ConstituencyData) => c.constituencyIdentifiers.slug,
   );
 };
 
-const getConstituenciesData = async (
-  cached: boolean = true,
-): Promise<ConstituencyData[]> => {
-  return cached
-    ? await getConstituenciesDataCached()
-    : await getConstituenciesDataUncached();
-};
-
-const getConstituenciesDataCached = unstable_cache(
-  // Cache data function
-  async () => getConstituenciesDataUncached(),
-  // Cache key
-  [`data/constituency.json.${serverRuntimeConfig.appVersion}`],
-  // Cache options
-  { revalidate: false }, // Cache will never refresh
-);
-
-const getConstituenciesDataUncached = async () => {
-  console.debug("constituencyData: fetching constituencies data");
-  const filePath = path.join(process.cwd(), "data", "constituency.json");
-  const fileContent = readFileSync(filePath, "utf8");
-  console.debug("constituencyData: fetched constituencies data from file");
-  return JSON.parse(fileContent);
+const getConstituenciesData = (cached: boolean = true): ConstituencyData[] => {
```

Comment:
Noice, way simpler. Can drop the `cached` arg as well now as it's redundant. It was optional anyway, and think it was only used in one or two places.